ci-connie.yml

```diff
@@ -135,9 +137,12 @@ post-processors:
     tag: "{{ user `docker_tag` }}"
     only:
     - docker
-  - type: artifice
-    files:
-    - tmp/docker-meta/.dumped
+
+- type: manifest
+  - output: manifest.json
+    strip_path: true
+    - inline:
+    - jq ".builds[].files[].name" manifest.json | xargs tar cfz artifacts.tgz
```

Comment:
I'm pretty sure this snippet (used elsewhere, too) is not valid YAML, and upon further reflection I don't think we need to use the `manifest` post-processor anyway, so backing out this change is probably what should be done 💙